### ISSUE TITLE
Invalid string escapes

### ISSUE THREAD

**Issue by nascheme** [dateutil/dateutil#361]
```
dateutil/parser.py:50: DeprecationWarning: invalid escape sequence \.
dateutil/tz/win.py:197: DeprecationWarning: invalid escape sequence \{
dateutil/tz/win.py:247: DeprecationWarning: invalid escape sequence \{
```

**Comment by nascheme** [comment]
Patch to fix.

[escapes.txt](https://github.com/dateutil/dateutil/files/922550/escapes.txt)

**Comment by jdufresne** [comment]
Will be fixed by #358.

**Comment by pganssle** [comment]
Fixed by #358.
